### PR TITLE
dashboard styling

### PR DESCRIPTION
**Pull request by mlippert**
## Description
Update the dashboard styling to follow the guidelines for scss styling that the team discussed

- move styles to the appropriate scss partials for the refactored dashboard code
- create sass variables to match the colors named in utils/colors.js
- remove unused package [react-sigma](https://github.com/dunnock/react-sigma) which was used when the influence chart used a network graph to convey its information.

## Motivation and context
Continuing dashboard refactor work including new style structure

## How has this been tested?
Tested locally on my machine

## Types of changes

- ✅ Refactor, code cleanup (non-breaking change which doesn't change functionality)

## Checklist:

- ✅ My code follows the code style of this project. **[REQUIRED]**
- ✅ All new and existing tests passed. **[REQUIRED]**

### CODE REVIEW

File: src/sass/abstracts/_variables.scss [hunk 1]
```diff
@@ -18,18 +18,39 @@
...
+$color-apricot:         #f2a466;
+$color-brightRed:       #f44336;
+$color-reddishPeach:    #f56b6b;
+$color-purpleWhite:     #f6f0fb;
+$color-redPunch:        #ff6384;
+$color-white:           #ffffff;
 
-$primary: $purple;
-$link: $purple;
+// BRAND COLORS (I've commented out the unused variables for now -mjl)
```
Comment (brecriffs) [inline src/sass/abstracts/_variables.scss]: I also decided to leave commented out styles for now. Seemed like the way to go.

File: src/sass/base/_base.scss [hunk 2]
```diff
@@ -15,18 +15,18 @@
...
 }
 
 html {
-    background-color: #f6f0fb;
+    background-color: $color-purpleWhite;
 }
 
 a:hover {
-    color: $purpleHover;
+    color: change-color($color-mauve, $alpha: 0.7);
```
Comment (brecriffs) [inline src/sass/base/_base.scss]: I didn't know about change-color. Very useful for adding alpha values to predefined colors.

**Comment by brecriffs** [comment]
15 min code review

**Comment by mlippert** [comment]
**Release** [1.2.0-dev.14](https://github.com/rifflearning/riff-rtc/releases/tag/1.2.0-dev.14)

**Pull request merged**